data/projects/documentation_extraction.md

```diff
@@ -0,0 +1,33 @@
+Code needs documentation. Most programming languages have some form of inline 
+documentation standard.
```

Comment:
This needs a little story as motivation. What Problem are we trying to fix? How would coala and its users benefit from the project?